Modules/Components/Niftyreg/include/selxItkToNiftiImage.hxx

```diff
@@ -174,7 +174,8 @@ ItkToNiftiImage< ItkImageType, NiftiPixelType >::Convert( typename ItkImageType:
     pixelContainer->ContainerManageMemoryOff();
   }
 
-  std::shared_ptr< nifti_image > ptr( output, nifti_image_free );
+  //std::shared_ptr< nifti_image > ptr(output, [](nifti_image* ptr){nifti_image_free(ptr); ptr = NULL; });
+  std::shared_ptr< nifti_image > ptr(output);
```

Comment:
This 'output' was allocated by a calloc call in Niftyreg\reg-io\nifti\nifti1_io.c, but shared_ptr just works OK with its default deleter, right?